x-pack/plugins/secops/public/components/drag_and_drop/droppable_wrapper.tsx

```diff
@@ -21,10 +21,48 @@ const getBackgroundColor = (theme: Theme): string =>
 
 const ReactDndDropTarget = styled.div<{ isDraggingOver: boolean; themeName: Theme }>`
   transition: background-color 0.7s ease;
-  min-height: 100px;
-  .euiPanel {
-    background-color: ${({ isDraggingOver, themeName }) =>
-      isDraggingOver ? '#f0f8ff' : getBackgroundColor(themeName)};
+  width: 100%;
+  height: 100%;
+  .flyout-overlay {
+    .euiPanel {
+      background-color: ${({ themeName }) => getBackgroundColor(themeName)};
+    }
+  }
+  ${({ isDraggingOver }) =>
+    isDraggingOver
+      ? `
+    .drop-and-provider-timeline {
+      &:hover {
+        background: repeating-linear-gradient(
+          -55deg,
+          rgb(52, 55, 65),
+          rgb(52, 55, 65) 10px,
+          rgb(245, 247, 250) 10px,
+          rgb(245, 247, 250) 20px
+        );
+      }
+    }
+  > div.timeline-drop-area {
+    background-color: rgb(245, 247, 250);
+    .provider-item-filter-container div:first-child{
+      transform: none !important;
+    }
+  }
+  .flyout-overlay {
+    .euiPanel {
+      background-color:  rgb(245, 247, 250);
+    }
+    + div {
+      display: none !important;
```

Comment:
Optional, I would add some comments about why we have to use `!important` when we do use it as down the road it's going to be tricky to unwind these when we need to.